region/tourian/main/Metroid Room 1.json

```diff
@@ -422,6 +427,15 @@
         }
       }
     },
+    {
+      "link": [2, 2],
+      "name": "Crystal Flash",
+      "requires": [
+        {"obstaclesCleared": ["A"]},
+        "h_canCrystalFlash"
+      ],
+      "note": ["Be sure to be at a safe distance from Rinkas before performing the Crystal Flash."]
```

Comment:
Node 2 looks like the free one.
Node 1 has a rinka at the door.

Reply:
Right, fixed now.